tests/test_flatten.py

```diff
@@ -0,0 +1,30 @@
+from geopandas import GeoDataFrame
+from pandas import DataFrame
+from pytest import raises
+from shapely.geometry import Point
+
+from crossfire import flatten
+
+DICT_DATA = [
+    {"answer": 42, "contextInfo": {"context1": "info1", "context2": "info2"}}
+]
+PD_DATA = DataFrame(DICT_DATA)
+geometry = [Point(4, 2)]
```

Comment:
Is this a constant? If so

```suggestion
GEOMETRY = [Point(4, 2)]
```